assets/catalog/parsed_plugin_list.js

```diff
@@ -3,7 +3,7 @@ myfcn(
   {
     "name": "Ping",
     "type": "Collector",
-    "description": "Collects Ping latency metrics",
+    "description": "No description available.",
```

Comment:
Is it possible to keep former description when there is already one in this file?